packages/main/src/Table.ts

```diff
@@ -72,6 +78,17 @@ type TableRowClickEventDetail = {
 	row: TableRow,
 };
 
+type TableMoveEventDetail = {
```

Comment:
You should also describe the `move` and `move-overs` events using `@event` decorator